jax/test_util.py

```diff
@@ -708,6 +708,9 @@ class JaxTestCase(parameterized.TestCase):
   # def tearDown(self) -> None:
   #   assert core.reset_trace_state()
 
+  def setUp(self):
+    core.skip_checks = False
```

Comment:
Did you want to keep this here, in addition to the FLAGS handling?

Reply:
I was planning to keep it. This will ensure that whenever we all run the tests they run with checks. The flag I thought would be useful for running non-tests, e.g., user's code with checks

Reply:
Ah, got it. Makes perfect sense!